app/experimenter/nimbus-ui/src/components/PageResults/TableMetricSecondary/index.test.tsx

```diff
@@ -88,4 +88,20 @@ describe("TableMetricSecondary", () => {
     expect(positiveBlock).not.toBeInTheDocument();
     expect(neutralBlock).not.toBeInTheDocument();
   });
+
+  it("shows expected count values", async () => {
```

Comment:
Does this need to be `async`?

Reply:
I believe it can be removed. I'll update